# ====== DO NOT MODIFY! AUTOGENERATED FROM AN ACTION DEFINITION ======

AlignedDrivingActionGoal action_goal
AlignedDrivingActionResult action_result
AlignedDrivingActionFeedback action_feedback
